src/animatediff/generate.py

```diff
@@ -1156,20 +1177,12 @@ def run_inference(
         region_condi_list=region_condi_list,
         interpolation_factor=1,
         is_single_prompt_mode=is_single_prompt_mode,
+        callback=callback,
+        callback_steps=output_map["preview_steps"],
```

Comment:
I would like this parameters( callback, callback_steps) to be set to a value only if "preview_steps" exists, otherwise it should be set to None.